server/sonar-web/src/main/js/apps/portfolio/components/App.tsx

```diff
@@ -152,13 +152,13 @@ export default class App extends React.PureComponent<Props, State> {
         </div>
 
         {subComponents != undefined &&
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') != should be !== [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1440)